### PR TITLE
Add 'update-seeker' view

### LINKED ISSUE #161: Add an seeker-update-details view
- Add a seeker-update-details view.
- Add Form classes to handle the details update.
- Add a template to update the details.

### PR DESCRIPTION
**Pull request by tamirho**
# Description
- Change the name of `ApartmentQualitiesUpdateForm` to `QualitiesForm` move it to `users/form.py` to re-use in multiple apps and to be in the right place that it should be.
- Move the logic of the `SeekerCreationForm` to the `SeekerUpdateForm` and use inheritance to re-use the code and the tests that were created by @Micha546  before. 
- Adding `update_seeker` view.
- Adding the `update-seeker` template to view the forms.

## Screenshots
Capture1
Capture2

## Manual Tests
- Register a new seeker and go to the `/seekers/update` route, change some fields(from both of the forms) and save it.
- Try to get into the `/seekers/update` route as an owner. Expecting to redirect to the home page.

### Issues closed by this PR
Fix #161

### CODE REVIEW

File: seekers/urls.py [hunk 1]
```diff
@@ -3,5 +3,6 @@
 
 
 urlpatterns = [
+    path('update', seeker_views.update_seeker, name='seeker-update'),
```
Comment (Micha546) [inline seekers/urls.py]: Can you add a / after the update?

Other than that looks great! 🔥

**Review by simzacks — APPROVED** [review]

**Comment by simzacks** [comment]
approved, but add the fix @Micha546 requested

File: seekers/urls.py (hunk 1 above)
Comment (tamirho) [reply seekers/urls.py]: done

**Comment by OrAsaf** [comment]
Need to resolve conflicts

**Comment by tamirho** [comment]
> Need to resolve conflicts
@1Oreo 

Fix that.
Please review again.

**Review by DanielMy777 — COMMENTED** [review]
👏 👏 👏

**Review by OrAsaf — APPROVED** [review]

**Pull request merged**